foundation/foundation-mda/src/main/resources/templates/integration-test/it.pipeline.spec.vm

```diff
@@ -1,6 +1,8 @@
 @pipeline
```

Comment:
i: The original stubbed integration test were incomplete and failing. Updated for now to be a more generic and empty stub.